### ISSUE TITLE
Make import name configurable

### ISSUE THREAD

**Issue by mxstbr** [styled-components/stylelint-processor-styled-components#109]
Right now this processor is hard-coded to only work with `styled-components` due to checking the import name here:

https://github.com/styled-components/stylelint-processor-styled-components/blob/f67b19276f6fc252af244d60d3e69fe8e8159a3b/src/utils/styled.js#L6-L7

We should add an option (`importName`? `library`? `package`?) to allow users to say that the name of the imported package is different so it can be used with `styled-jss`, `emotion`, etc.

It'd be a shame if all those libraries had to build their own processor, when really they only have to change that one string and it'd work. I don't think we should promise any particular level of support, since all those libraries have slightly diverging APIs, but having basic support would be prettttttty great. (and also awesome to get out with v1.0)

**Comment by mxstbr** [comment]
Ohh I just now realised that this'll already work as we only check for `styled.x` calls! This code lints:

```JS
import styled from 'emotion';

const Div = styled.div`
  color: blue;
`
```

It only fails if you use a differing import name, that's what that check is for!

```JS
import emotion from 'emotion';

const Div = emotion.div`
  color: blue;
`
```
